src/net.cpp

```diff
@@ -2214,7 +2215,7 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
             }
         }
 
-        if (InactivityCheck(*pnode)) pnode->fDisconnect = true;
+        if (InactivityCheck(*pnode, now)) pnode->fDisconnect = true;
```

Comment:
nit, feel free to ignore: given that we allow some nontrivial amount of time to pass between the variable initialization and usage, maybe `now` is not the best name for it. What about `time_at_start_of_loop` or something like that?

Reply:
I'm not sure I'd say "we allow some nontrivial amount of time to pass" -- it's probably a bug if that were to actually happen?